src/js/components/stock-movement-wizard/request/CreateStockMovement.jsx

```diff
@@ -253,6 +301,20 @@ class CreateStockMovement extends Component {
       .catch(() => this.props.hideSpinner());
   }
 
+  fetchAvailableApprovers() {
+    return userApi.getUsersOptions({
+      params: { roleTypes: RoleType.ROLE_REQUISITION_APPROVER },
+    })
+      .then((response) => {
+        this.setState({
+          availableApprovers: response.data.data,
+        }, () => this.props.hideSpinner());
+
+        return response.data.data;
+      })
+      .catch(() => this.props.hideSpinner());
```

Comment:
let's maybe add `.finally` here that would hide the spinner instead of hiding in both in `.catch` and in `.then`?

Reply:
good idea, I was just copy/pasting the previous fetch methods in the component and followed the "convention"